src/mgr.sol

```diff
@@ -192,54 +188,49 @@ contract TinlakeManager {
 
     // moves the rwaToken into the vault
     // requires that mgr contract holds the rwaToken
-    function lock(uint wad) public operatorOnly {
+    function lock(uint256 wad) public auth {
         require(vat.live() == 1, "TinlakeManager/mkr-in-ES");
+        rwaToken.approve(address(urn), uint256(wad));
```

Comment:
Doing this here means we can call `file("rwaToken", addr)` before the `urn` is set. Slightly more robust